src/core/tests/type_relaxed_copy.cpp

```diff
@@ -42,7 +42,7 @@ TEST_F(TypeRelaxedThreading, TypeRelaxedCloning) {
     auto inp2 = std::make_shared<op::v0::Parameter>(element::i8, PartialShape{-1, -1, -1, -1});
 
     auto matMulRelaxed = std::make_shared<ov::op::TypeRelaxed<ov::op::v0::MatMul>>(
-        *as_type_ptr<op::v0::MatMul>(ngraph::builder::makeMatMul(inp1_f32, inp2_f32, false, false)),
+        *std::make_shared<ov::op::v0::MatMul>(inp1_f32, inp2_f32, false, false),
```

Comment:
`shared_ptr` is not needed here.
```suggestion
        ov::op::v0::MatMul{inp1_f32, inp2_f32, false, false},
```